### PR TITLE
feat(interactive-activate): description and exit instructions

### LINKED ISSUE #724: flox activate tells you how to exit
# Acceptance Criteria
**Given** I have an environment in my current directory
**When** I `flox activate`
**Then** I see a loading message with a spinner:
```
:. Getting ready to use environment project1 at /path/to/env/project1
```
**And then** when it is completed I see the loading message is replaced with a confirmation message:
```
✅ You are now using the environment ‘project1’ at /path/to/env/project1 
To stop using this environment, type ‘exit’
```

### PR DESCRIPTION
**Pull request by ysndr**
closes #724

### CODE REVIEW

File: cli/flox/src/commands/environment.rs [hunk 1]
```diff
@@ -436,7 +436,7 @@ impl Activate {
             environment.activation_path(&flox)?
         } else {
             Dialog {
-                message: &format!("Building environment '{prompt_name}'..."),
+                message: &format!("Getting ready to use environment '{prompt_name}'..."),
```
Comment (mkenigs) [inline cli/flox/src/commands/environment.rs:439]: suggestion: ticket asks for name at location format, not just name

File: cli/flox/src/commands/environment.rs [hunk 2]
```diff
@@ -584,6 +584,13 @@ impl Activate {
         }
 
         debug!("running activation command: {:?}", command);
+
+        if self.run_args.is_empty() {
+            let message = formatdoc! {"
+                ✅  You are now using the environment {now_active}
+                To stop using this environment, type 'exit'"};
```
Comment (mkenigs) [inline cli/flox/src/commands/environment.rs:591]:
```suggestion
                ✅  You are now using the environment {now_active}.
                To stop using this environment, type 'exit'"};
```

**Review by mkenigs — CHANGES_REQUESTED** [review]

File: cli/flox/src/commands/environment.rs (hunk 1 above)
Comment (ysndr) [reply cli/flox/src/commands/environment.rs:439]:
```suggestion
                message: &format!("Getting ready to use environment '{now_active}'..."),
```

File: cli/flox/src/commands/environment.rs [hunk 3]
```diff
@@ -436,7 +436,7 @@ impl Activate {
             environment.activation_path(&flox)?
         } else {
             Dialog {
-                message: &format!("Building environment '{prompt_name}'..."),
+                message: &format!("Getting ready to use environment '{now_active}'..."),
```
Comment (mkenigs) [inline cli/flox/src/commands/environment.rs:439]:
```suggestion
                message: &format!("Getting ready to use environment {now_active}..."),
```

**Review by mkenigs — APPROVED** [review]

**Review by mkenigs — APPROVED** [review]

**Pull request merged**